server/routes/public.js

```diff
@@ -28,6 +28,57 @@ router.get('/login', async (req, res) => {
     res.send(user);
 });
 
+<<<<<<< Updated upstream
```

Comment:
Hi Nancy,
I would suggest removing lines 31 to 34 as these lines will interfere with the code execution. I think they were left behind as a result of resolving the merge conflicts.

Reply:
Hi Amy,
Nice catch!
My action: I've removed lines 31-34!